src/styles/components/HeaderMenuDropDownStyles.js

```diff
@@ -47,7 +47,16 @@ const MenuDropDownStyle = {
   altIconText: {
     fontSize: 20
   }
+}
 
+const MenuDropDownStyleHeader = {
```

Comment:
I don't understand how this fixes the problem. So you had to remove `flex` and `justifyContent` to get this to work? Does this affect iOS?